### PR TITLE
fix #275539: problems installing on linux

### PR DESCRIPTION
**Pull request by jthistle**
Resolves: https://musescore.org/en/node/275539

This solves problems installing for me, and for [this user](https://musescore.org/en/node/275539#comment-850970).

### CODE REVIEW

File: mscore/CMakeLists.txt [hunk 1]
```diff
@@ -618,7 +618,7 @@ else (MINGW)
 
    if ( NOT MSVC )
 ## install qwebengine core
-      if (NOT APPLE AND USE_WEBENGINE)
+      if (NOT APPLE AND NOT LN_EXECUTABLE AND USE_WEBENGINE)
```
[the remark this answers is not in the document]
Comment (jthistle) [reply mscore/CMakeLists.txt]: LN_EXECUTABLE is true when musescore is built on a Linux machine, hence 'LN'.

**Comment by jthistle** [comment]
I actually can't work out how to enable/view MuseScore connect, so maybe it doesn't work with this fix :/

**Comment by anatoly-os** [comment]
MuseScore connect is the right widget in the start center. There are several controls there like scores search and carousel.

**Comment by anatoly-os** [comment]
Are you sure? I mean did you check on clean environment? This step installs QtWebEngineProcess which is required by QtWebEngine to show web pages in MuseScore Connect. If this patch is applied, there will be no place to copy the process file to run chrome web engine. Could you please clean the bin folder first and rebuild MuseScore again with this fix? If QtWebEngineProcess is there, I will need to check other places where QtWebEngineProcess is copied on Linux...

**Comment by jthistle** [comment]
I cleared my /usr/local/bin, did `make clean`, then `make debug` followed by `sudo make installdebug`. The MuseScore Connect display still works. To be honest, I'm not 100% sure how the install functions without this bit, but I've experienced no issues from my fix.

File: mscore/CMakeLists.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (dmitrio95) [reply mscore/CMakeLists.txt]: If I understand this correctly, we check that we are not using MSVC, then check that we are not on Mac OS and now we intend to check that we are not on a Linux-based system, don't we? So what is left? MinGW builds? And do we need to install QtWebEngine then at all?
By the way LN in `LN_EXECUTABLE` doesn't seem to stand for "Linux" though this variable setting may coincide with UNIX-like environments: https://github.com/musescore/MuseScore/blob/559ee89127fe0d3719a9a5d3d94ac1ee876ababb/CMakeLists.txt#L653

**Comment by jthistle** [comment]
Closing, because of a complete fix by @dmitrio95 : #4224

**Pull request closed without merging**